security/db_router.py

```diff
@@ -31,3 +31,33 @@ def allow_relation(self, obj1, obj2, **hints):
 
     def allow_migrate(self, db, app_label, model_name=None, **hints):
         return True
+
+
+class MultipleDBSecurityLoggerRouter:
+    """
+    A router controlling tha all that read and write database operations will be performed via mirror without atomic.
+    """
+
+    route_app_labels = [
+        'security', 'debug_toolbar_log'
+    ]
+
+    def _db_for_read_or_write(self, model):
+        if model._meta.app_label in self.route_app_labels:
+            return settings.LOG_DB_NAME
+        return None
```

Comment:
Why don't you return the default DB here?

Reply:
look to the documentation. There can be more routers. Your router should decide only about your tables but other tables are left for next defined router